### PR TITLE
Created atan2() reference function for php

### PR DESCRIPTION
**Pull request by arisdelacruz**
### Description

This is an attempt to create the ATAN2() reference function for php. Motivation comes from the aspiration to eventually become a regular contributor to codecademy docs.

### Type of Change

- Adding a new entry

### Checklist

- [ /] All writings are my own.
- [ /] My entry follows the Codecademy Docs style guide.
- [ /] My changes generate no new warnings.
- [ /] I have performed a self-review of my own writing and code.
- [ /] I have checked my entry and corrected any misspellings.
- [ /] I have made corresponding changes to the documentation if needed.
- [ /] I have confirmed my changes are not being pushed from my forked `main` branch.
- [ /] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [ /] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/atan2/atan2.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+Title: 'atan2()'
+Description: 'The atan2() function returns the arc tangent of y/x, in radians.'
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan2/atan2.md:3]:
```suggestion
Description: 'Returns the arc tangent of y/x, in radians.'
```

File: content/php/concepts/math-functions/terms/atan2/atan2.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+The **`atan2()`** function returns the arc tangent of y/x, in radians.
+
+## Syntax
+
+```pseudo
+atan2(y,x);
+```
+
+The `atan2()` function has two (2) parameters. It returns the arc tangent of y/x, in radians. Where x and y are the coordinates of a point (x,y). Both specify a positive or negative number.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan2/atan2.md:25]:
```suggestion
The `atan2()` function has two (2) parameters. It returns the arc tangent of *y/x*, in radians. Where `x` and `y` are the coordinates of a point (x, y). Both specify a positive or negative number.
```

File: content/php/concepts/math-functions/terms/atan2/atan2.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+  - 'Methods'
+  - 'Arithmetic'
+  - 'Numbers'
+  - 'Functions'
+CatalogContent:
+  - 'learn-php'
+  - 'paths/computer-science'
+---
+
+The **`atan2()`** function returns the arc tangent of y/x, in radians.
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan2/atan2.md:17]:
```suggestion
The **`atan2()`** function returns the arc tangent of *y/x*, in radians.
```

File: content/php/concepts/math-functions/terms/atan2/atan2.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+## Syntax
+
+```pseudo
+atan2(y,x);
+```
+
+The `atan2()` function has two (2) parameters. It returns the arc tangent of y/x, in radians. Where x and y are the coordinates of a point (x,y). Both specify a positive or negative number.
+
+## Codebyte Example
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan2/atan2.md:27]:
```suggestion
## Example
```

File: content/php/concepts/math-functions/terms/atan2/atan2.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+Using `atan2()` to return the arc tangent of (-10,8):
+
+```codebyte/php
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan2/atan2.md:31]:
```suggestion
```php
```

File: content/php/concepts/math-functions/terms/atan2/atan2.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+<?php
+  echo atan2(-10,8);
+?>
+```
+
+The output is -0.89605538457134
```
Comment (caupolicandiaz) [inline content/php/concepts/math-functions/terms/atan2/atan2.md:37]:
```suggestion
This results in the following output:

```shell
 -0.89605538457134
 ```

```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@arisdelacruz I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by arisdelacruz** [comment]
@caupolicandiaz 

done

thanks

**Review by caupolicandiaz — APPROVED** [review]
@arisdelacruz Looks good. I'm passing this on to a second review.

**Review by SSwiniarski — APPROVED** [review]
@arisdelacruz Looks good to go.

**Comment by SSwiniarski** [comment]
@arisdelacruz Congrats on the new entry! Live link here: https://www.codecademy.com/resources/docs/php/math-functions/atan2

image

**Pull request merged**